ci/jenkins/stop-jobs.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+PULL_REQUEST_ID=""
+JENKINS_URL=""
+DEFAULT_WORKDIR="/var/lib/jenkins"
+
+_usage="Usage: $0 [--pull-request] [--jenkins]
+
+        --pull-request           Unique Pull Request id required to find the correct stale jobs need to abort.
+        --jenkins                Using jenkins url fetch the information about jobs through REST API."
+
+function print_usage {
+    echoerr "$_usage"
+}
+
+function print_help {
+    echoerr "Try '$0 --help' for more information."
+}
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+    --pull-request)
+    PULL_REQUEST_ID="$2"
+    shift 2
+    ;;
+    --jenkins)
+    JENKINS_URL="$2"
+    shift 2
+    ;;
+    -h|--help)
+    print_usage
+    exit 0
+    ;;
+    *) 
+    echoerr "Unknown option $1"
+    exit 1
+    ;;
+esac
+done
+
+function abort_queue_jobs {
+    echo "====== checking for capv stale waiting jobs on PR ${PULL_REQUEST_ID} ======"
+    capv_job_name=$1
+    length_of_queue=$(curl -s -XGET "${JENKINS_URL}queue/api/json" | jq '[.[]][2]' | jq length)
+
+    for (( j=0; j<$length_of_queue; j++ )); do
+        job_name=$(curl -s -XGET "${JENKINS_URL}queue/api/json" | jq "[.[]][2][$j].task.name" | tr -d '"') 
+        if [[ "$job_name" == "$capv_job_name" ]]; then
+            # fetch the waiting job_id and waiting job pr_id
```

Comment:
use the actual meaning of variable instead of the variable name in comments, i.e you can write fetch the waiting Job id, and ...